examples/RAmPART_examples/main.m

```diff
@@ -0,0 +1,72 @@
+% This file shows how to execute several PESTO examples using the
+% RAmPART-sampler. The options are set in selectRun.m and the results are
+% getting stored in [pwd filesep 'results'].
+
+% Add pathes of PESTO and the examples
+path(pathdef)
+addpath(['..' filesep '..']);
+addpath([pwd filesep 'Banana']);
+addpath([pwd filesep 'Ring']);
+addpath([pwd filesep 'Gauss']);
+addpath([pwd filesep 'Bachmann_JakStat']);
+addpath([pwd filesep 'JakStat']);
+addpath([pwd filesep 'RafMekErk']);
+addpath([pwd filesep 'mRNA_Transfection']);
+
+% Add you AMICI path here (necessary for some of the examples)
+addpath('C:\Users\benjamin\Home\GEYAW_Tools\AMICI')
```

Comment:
better: check if amiwrap exists on path and if not ask to add AMICI folder to the path. Nobody who runs the examples script will change that line beforehand.